src/instruments/src/SD/Pages/Apu/Apu.tsx

```diff
@@ -0,0 +1,521 @@
+    const [busTieContactor2Closed] = useSimVar('L:A32NX_ELEC_CONTACTOR_11XU2_IS_CLOSED', 'Bool');
+
+    const [apuGenLoad] = useSimVar('L:A32NX_ELEC_APU_GEN_1_LOAD', 'Percent', 500);
+    const [apuGenLoadNormalRange] = useSimVar('L:A32NX_ELEC_APU_GEN_1_LOAD_NORMAL', 'Bool', 750);
+
+    const [apuGenVoltage] = useSimVar('L:A32NX_ELEC_APU_GEN_1_POTENTIAL', 'Volts', 500);
+    const [apuGenPotentialNormalRange] = useSimVar('L:A32NX_ELEC_APU_GEN_1_POTENTIAL_NORMAL', 'Bool', 750);
+
+    const [apuGenFreq] = useSimVar('L:A32NX_ELEC_APU_GEN_1_FREQUENCY', 'Hertz', 500);
+    const [apuGenFreqNormalRange] = useSimVar('L:A32NX_ELEC_APU_GEN_1_FREQUENCY_NORMAL', 'Bool', 750);
+
+    enum apuGenState {STANDBY, OFF, ON}
+    const [currentApuGenState, setCurrentApuGenState] = useState(apuGenState.STANDBY);
+
+    const [apuMasterPbOn] = useSimVar('L:A32NX_OVHD_APU_MASTER_SW_PB_IS_ON', 'Bool', 500);
+    const [apuGenPbOn] = useSimVar('A:APU GENERATOR SWITCH', 'Boolean', 750);
+    const [apuAvail] = useSimVar('L:A32NX_OVHD_APU_START_PB_IS_AVAILABLE', 'Bool', 1000);
+
+    useEffect(() => {
+        if (!apuMasterPbOn && !apuAvail /* FIXME: and apu master switch is valid and apu gen push buttion is valid */) {
+            setCurrentApuGenState(apuGenState.STANDBY);
+        } else if (!(currentApuGenState === apuGenState.STANDBY) && !apuGenPbOn /* FIXME: and apu master switch valid and apu gen pb valid */) {
+            setCurrentApuGenState(apuGenState.OFF);
+        } else {
+            setCurrentApuGenState(apuGenState.ON);
+        }
+    }, [apuMasterPbOn, apuAvail]);
+
+    return (
+        <>
+            <SvgGroup x={x} y={y}>
+                {(apuContactorClosed && (busTieContactor1Closed || busTieContactor2Closed))
+                    && (
+                        <SvgGroup x={42} y={-28}>
+                            <polygon points="0,20 8,5 16,20" className="Circle" />
+                        </SvgGroup>
+                    )}
+                {currentApuGenState !== apuGenState.STANDBY
+                && <rect className="Box" width={100} height={111} />}
+
+                <text
+                    x={50}
+                    y={20}
+                    className={`Center FontNormal
+                    ${currentApuGenState === apuGenState.OFF
+                         || (!(apuGenPotentialNormalRange && apuGenLoadNormalRange && apuGenFreqNormalRange) && currentApuGenState === apuGenState.ON) && ' Amber'}`}
+                >
+                    APU GEN
+                </text>
+
+                {currentApuGenState === apuGenState.OFF
+                    && <text x={50} y={70} className="Center FontNormal">OFF</text>}
+
+                {currentApuGenState === apuGenState.ON
+                    && (
+                        <>
+                            <SvgGroup x={60} y={55}>
+                                {/* FIXME: replaced by amber crosses when information is not available */}
```

Comment:
**thought**: These "amber crosses" FIXME comments are more realistic to achieve soon, as we already have the means of writing ARINC429 data and thus could implement this later.

**general remark**: `FIXME` is a bit misnamed, as there's nothing wrong with the implementation: there are no bugs, just missing features. I'd write `TODO`. I'd also like to note that `TODO`s should be added sparingly, and they should describe exactly what needs to be done, to ensure it is clear for anyone else in the future. E.g. "Add amber crosses when information is not available, by implementing component _x_ and passing data from the APU's ECB in ARINC 429 format".